graviti/schema/builtin.py

```diff
@@ -5,20 +5,53 @@
 """The Portex builtin types."""
 
 
-from typing import Any, Iterable, List, Mapping, Optional, Sequence, Tuple, Union, overload
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Iterable,
+    List,
+    Mapping,
+    Optional,
+    Sequence,
+    Tuple,
+    Union,
+    overload,
+)
+
+import pyarrow as pa
 
 import graviti.schema.ptype as PTYPE
 from graviti.schema.base import PortexType
 from graviti.schema.field import Field, Fields
 from graviti.schema.package import packages
 from graviti.schema.param import Param, Params, param
+from graviti.schema.pyarrow import BuiltinExtension
+
+if TYPE_CHECKING:
+    from pyarrow.lib import DataType as paDataType
 
 builtins = packages.builtins
 
+_TYPE_TO_PYARROW = {
```

Comment:
what is this constant for？
python type to pyarrow type
or
 Portex type to pyarrow type?